game/gameboard_view.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+from gameboard import GameBoard
+
+class GameBoardView(object):
+
+    def show(self, board):
+        """Print the board of a given gameboard in a nice format"""
+        print(chr(27) + "[2J")
+        for row in board.retrieve_matrix():
```

Comment:
Added a new method: get_matrix_after_doing, where I return a matrix (list of lists), which is an image of the original matrix after applying a given function